### PR TITLE
Add binutils version

### PR DESCRIPTION
**Pull request by ice1000**
Signed-off-by: ice1000 <ice1000kotlin@foxmail.com>

See #274

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ This project is still under development. The following features with the check m
 
 - CMake >= 3.8.0
 - Rust >= 1.19.0
+- binutils >= 2.22
```
Comment (BusyJay) [inline README.md]: I don't think it's necessary. And it's usually a dependency of gcc.

Comment (ice1000) [reply README.md]: How about requiring a newer version of gcc?
Actually I don't know the corresponding gcc version.

Comment (BusyJay) [reply README.md]: Sounds more reasonable to me. Whatever works for grpc/grpc.

Comment (ice1000) [reply README.md]: According to https://grpc.io/about/

gcc **4.4, 4.6**, 5.3 are supported while in #274 people are using gcc 4.8 and 4.9.
It's like the version of binutils is independent to gcc.

**Pull request merged**